lib/datadog/appsec/actions_handler/stack_trace/collector.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+module Datadog
+  module AppSec
+    module ActionsHandler
+      module StackTrace
+        # Represent a stack trace with its id and message in message pack
+        module Collector
+          class << self
+            def collect(locations)
+              return [] if locations.nil? || locations.empty?
+
+              skip_frames = skip_frames(locations.size)
+              frames = []
```

Comment:
I would suggest the same here and everywhere - let's try to have consistent naming, if we call those `frames`, let's not call them `locations` (or the other way around)